PCbuild/pythoncore.vcxproj

```diff
@@ -651,7 +651,11 @@
     <ClCompile Include="..\Python\structmember.c" />
     <ClCompile Include="..\Python\symtable.c" />
     <ClCompile Include="..\Python\sysmodule.c">
-      <PreprocessorDefinitions>VPATH="$(PyVPath)";%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>
+        VPLATFORM="$(ArchName)";
```

Comment:
Why not "ARCH_NAME" or "ARCHITECTURE"?

Reply:
`ArchName` is a variable defined in `python.props`.

Reply:
Sorry, I was thinking at `VARCH_NAME` or `VARCH` variable names.

Reply:
Oh sorry, I misunderstood you. I see.

Reply:
I have made changes, Please reviewer this changes.